code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -142,6 +142,49 @@
 		GLOB.pre_setup_antags -= changeling
 	return TRUE
 
+//////////////////////////////////////////////
+//                                          //
+//              ELDRITCH CULT               //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/heretics
+	name = "Heretics"
+	antag_flag = ROLE_CULTIST
+	antag_datum = /datum/antagonist/heretic
+	protected_roles = list("Prisoner","Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	restricted_roles = list("AI", "Cyborg")
+	required_candidates = 1
+	weight = 3
+	cost = 15
+	scaling_cost = 10
```

Comment:
Too cheap. With an average threat of 50, this ruleset can spawn a full set at 45 threat fairly often. Up the cost somewhat to reduce the chances that it maxes out on top of something else. 20 and 15  is more palatable.